restcomm/restcomm.interpreter/src/main/java/org/restcomm/connect/interpreter/rcml/Parser.java

```diff
@@ -152,6 +162,47 @@ private Tag next() throws LimitExceededException{
         return null;
     }
 
+	///////////////////////////////////////////// new class -- validate
+    private boolean validation(Tag tag) {
+    	boolean ValidationPass=true;
+    	AttributeValidator AttValidator=new Validator();
+    	
+    	switch (tag.name()) {
```

Comment:
if we had a specialized validator class impl for each tag with a single API, we dont need this switch statement. We jsut need an static map with all verb validators, indexes by tagname, and ocne located call the unique validate method of the proper imp class